tutorials/not_so_obvious_python_stuff.ipynb

```diff
@@ -390,7 +377,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "\"It often comes as a big surprise for programmers to find (sometimes by way of a hard-to-reproduce bug) that, unlike any other time value, midnight (i.e. `datetime.time(0,0,0)`) is False. A long discussion on the python-ideas mailing list shows that, while surprising, that behavior is desirable—at least in some quarters.\"  \n",
+    "\"It often comes as a big surprise for programmers to find (sometimes by way of a hard-to-reproduce bug) that, unlike any other time value, midnight (i.e. `datetime.time(0,0,0)`) is False. A long discussion on the python-ideas mailing list shows that, while surprising, that behavior is desirable — at least in some quarters.\"  \n",
```

Comment:
Looks good, thx! 
I think em-dashes are used in so many different ways that there is no standard way to do it. Generally, I prefer a white space before and after an em-dash or "--" workaround.

Reply:
I'm glad everything is looking fine! if there is anything i need to go back and fix, please tell me! other than that, the notebook should be good to pull 👍